packages/node/src/ContentOptionsBuilder.ts

```diff
@@ -0,0 +1,51 @@
+import type { Response } from 'got'
+
+/* Optional Parameters? */
+export type ContentOptions = {
+  page: string,
+  contentType: string,
+  returnType?: 'all' | 'one',
+  filter: Record<string, boolean | string | number>,
+  order: Record<string, 'ASC' | 'DESC'>,
+  limit: number
+}
+
+class ContentOptionsBuilder {
+  private options: Partial<ContentOptions>
```

Comment:
We can remove `Partial` in several places, if we make filter, order, limit, page optional (`?`)